src/Engine/Graphics/LightmapBuilder.cpp

```diff
@@ -50,13 +50,13 @@ void DrawLightsDebugOutlines(
  *
  * @return                              Return 32bit colour ARGB.
  */
-int GetActorTintColor(int max_dimm, int min_dimm, float distance, int bNoLight, RenderBillboard *pBillboard) {
+Color GetActorTintColor(int max_dimm, int min_dimm, float distance, int bNoLight, RenderBillboard *pBillboard) {
     int dimminglevel = 0;
 
     if (uCurrentlyLoadedLevelType == LEVEL_INDOOR)
-        return 8 * (31 - max_dimm) | ((8 * (31 - max_dimm) | ((31 - max_dimm) << 11)) << 8);
+        return Color(8 * (31 - max_dimm), 8 * (31 - max_dimm), (31 - max_dimm) << 3); // TODO(captainurist): why << 3 ???
```

Comment:
<<3 is  x8 
other components have the same

Reply:
Oh =_=